dateparser/calendars/__init__.py

```diff
@@ -1,6 +1,6 @@
 # coding: utf-8
```

Comment:
Unless you prefer to do it in a later changeset, this may be a good chance to get rid of these encoding comments as well.

And that way we don‘t need to discuss whether the empty lines between those and the first import line should be 1 or 2 :slightly_smiling_face:

Reply:
YeahYeah, this will be handled in another PR: #758 :+1: